### DISCUSSION TITLE
Error while trying to transcript

### DISCUSSION THREAD

**Discussion by shahidirfan100** [openai/whisper#893] [Show and tell]
C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\transcribe.py:79: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")
Traceback (most recent call last):
  File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\main.py", line 4, in <module>
    result = model.transcribe("audio.mp3")
  File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\transcribe.py", line 85, in transcribe
    mel = log_mel_spectrogram(audio)
  File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\audio.py", line 111, in log_mel_spectrogram
    audio = load_audio(audio)
  File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\audio.py", line 42, in load_audio
    ffmpeg.input(file, threads=0)
  File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\site-packages\ffmpeg\_run.py", line 313, in run
    process = run_async(
  File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\site-packages\ffmpeg\_run.py", line 284, in run_async
    return subprocess.Popen(
  File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1440, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

**Comment by glangford** [comment]
Do you have `ffmpeg` installed? `whisper` requires it.

**Reply by LaoshuBaby** [reply]
Sure, I meet with this problem, I configured PATH to downloaded ffmpeg binary but failed, then I install it with choco, then whisper work correctly. ~~(but still using FP32 because I only have a core graphics with AMD CPU)~~

**Comment by doveg** [comment]
As the log says, you're looking for a file called "audio.mp3" in your working directory.
I'm afraid you'll have to check it in the directory you pass in.

By the way, "using FP32 instead" means whisper will only use the CPU for the task, if you want to take advantage of the GPU, check out the PyTorch version for "cuda".

**Comment by sivakumar41** [comment]
> C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\transcribe.py:79: UserWarning: FP16 is not supported on CPU; using FP32 instead warnings.warn("FP16 is not supported on CPU; using FP32 instead") Traceback (most recent call last): File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\main.py", line 4, in result = model.transcribe("audio.mp3") File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\transcribe.py", line 85, in transcribe mel = log_mel_spectrogram(audio) File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\audio.py", line 111, in log_mel_spectrogram audio = load_audio(audio) File "C:\Users\Shahid\Desktop\Python Scripts\whisper-main\whisper\audio.py", line 42, in load_audio ffmpeg.input(file, threads=0) File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\site-packages\ffmpeg_run.py", line 313, in run process = run_async( File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\site-packages\ffmpeg_run.py", line 284, in run_async return subprocess.Popen( File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 971, in **init** self._execute_child(args, executable, preexec_fn, close_fds, File "C:\Users\Shahid\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1440, in _execute_child hp, ht, pid, tid = _winapi.CreateProcess(executable, args, FileNotFoundError: [WinError 2] The system cannot find the file specified

Did you solved it?

**Reply by glangford** [reply]
* https://github.com/openai/whisper/discussions/61#discussioncomment-6459932
* https://github.com/openai/whisper/discussions/109

**Reply by sivakumar41** [reply]
The file is in the correct position.
I'm not able to install all the dependencies.
That may be the reason for the error

Can you guide me to install all the required libraries in the windows operating system.
Thank you
